flakeheaven/_cli.py

```diff
@@ -44,7 +44,7 @@ def entrypoint(argv: List[str] = None) -> NoReturn:
 
 
 def flake8_entrypoint(argv: List[str] = None) -> NoReturn:
-    """Entrypoint with the same behavior as flake8 (flake8helled)
+    """Entrypoint with the same behavior as flake8 (flake8heavened)
```

Comment:
I think we'd also need to update `pyproject.toml`:
```
flake8helled = "flakeheaven:flake8_entrypoint"
```
to
```
flake8heavened = "flakeheaven:flake8_entrypoint"
```

Reply:
Good spot.